lnbits/core/templates/admin/_tab_server.html

```diff
@@ -75,7 +75,7 @@ <h6 class="q-my-none">Server Management</h6>
             clearable
             hint="Default currency for accounting"
             label="Currency"
-            :options="formData.lnbits_allowed_currencies.length ? formData.lnbits_allowed_currencies : {{ currencies }}"
+            :options="formData.lnbits_allowed_currencies && formData.lnbits_allowed_currencies.length ? formData.lnbits_allowed_currencies : {{ currencies }}"
```

Comment:
- why not use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)?
```js
formData.lnbits_allowed_currencies?.length
```